pkg/controller/operator.go

```diff
@@ -268,7 +268,7 @@ func (s *Operator) Run(ctx context.Context, controller *controllercmd.Controller
 		return fmt.Errorf("unable to set initial cluster status: %v", err)
 	}
 
-	scaController := sca.New(ctx, kubeClient.CoreV1(), configAggregator, insightsClient)
+	scaController := sca.New(ctx, kubeClient.CoreV1(), configAggregator, insightsClient, gatherProtoKubeConfig)
```

Comment:
Is this required? :thinking: We are already passing `kubeClient.CoreV1()` so I guess we could use it for listing Nodes.

Reply:
I didn't realise! Reverted this change and refactored the gathered function to use that client instead of creating a new one